Makefile

```diff
@@ -116,6 +115,11 @@ _verify_scss_lint:
 # HACK: Use backticks rather than xargs because xargs swallow exit codes (everything becomes 1 and stoopidly scss-lint exits with 1 if warnings, 2 if errors)
 	@if [ -e .scss-lint.yml ]; then { scss-lint -c ./.scss-lint.yml `$(call GLOB,'*.scss')`; if [ $$? -ne 0 -a $$? -ne 1 ]; then exit 1; fi; $(DONE); } fi
 
+
+MSG_PUBLIC_IGNORE = "Please ignore built assets individually - globs and entire folders not allowed"
+_verify_assets:
+	@if grep -Eq '^\/public\/(.*\*|$$)' .gitignore; then (echo $(MSG_PUBLIC_IGNORE) && exit 1); fi
```

Comment:
I think the regex should be… `public(\/(.*\*||$$))?`

probably could put the `-E` option into `IS_GIT_IGNORED` then this could be……

```
@if $(call IS_GIT_IGNORED,'^\/public\/(.*\*|$$)'); then 
```

also bit worried if this is going to be a bit disruptive.  i suspect there may be quite a few files that go into `public` (e.g. `asset-hashes.json`)